integration/core/test_controller.py

```diff
@@ -2,18 +2,28 @@
 import os
 import grpc
 import pytest
+import tempfile
```

Comment:
Can we move this to the `integration/instance`? Core test doesn't mean to create/delete processes.

Reply:
Sure, I just updated the PR.